go-controller/pkg/ovn/namespace.go

```diff
@@ -218,7 +236,60 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 	}
 	defer nsInfo.Unlock()
 
-	annotation := newer.Annotations[hotypes.HybridOverlayExternalGw]
+	var annotation, oldAnnotation string
+	annotation = newer.Annotations[routingExternalGWsAnnotation]
+	oldAnnotation = old.Annotations[routingExternalGWsAnnotation]
+	if annotation != oldAnnotation {
+		var stderr string
+		var err error
+		for k, v := range nsInfo.podExternalRoutes {
```

Comment:
maybe just use some real names for k,v here to make it more obvious what they are.

Reply:
done